unofficial_irods_documentation/mkdocs.yml

```diff
@@ -25,3 +25,5 @@ nav:
     #   - "Compound Trees"
     #   - "File system resources"
     #   - "S3 resources"
+markdown_extensions:
+  - admonition: {}
```

Comment:
Is that [this](https://squidfunk.github.io/mkdocs-material/reference/admonitions/) - what's it used for?

Reply:
Yes. I don't see where I used an one. I must have removed it. I'll remove the extension.